MAUI/Maps/shape-sublayer.md

```diff
@@ -56,7 +56,7 @@ N> Refer the `MapShapeLayer`, for adding shape layer in `SfMaps`.
 
 ## Color and stroke color
```

Comment:
The note section in the above line, API reference is not included.

N> Refer the `MapShapeLayer`, for adding shape layer in `SfMaps`.